pkg/pipeline/validation/task_validations.go

```diff
@@ -1,58 +1,102 @@
 package validation
 
 import (
-	"fmt"
+	errors2 "errors"
 	"strings"
 
+	operatorDefaults "github.com/equinor/radix-operator/pkg/apis/defaults"
 	"github.com/equinor/radix-tekton/pkg/defaults"
+	"github.com/pkg/errors"
 	pipelinev1 "github.com/tektoncd/pipeline/pkg/apis/pipeline/v1"
 	corev1 "k8s.io/api/core/v1"
 )
 
+var (
+	ErrEmptyStepList             = errors.New("step list is empty")
+	ErrSecretReferenceNotAllowed = errors.New("references to secrets are not allowed")
+	ErrRadixVolumeNameNotAllowed = errors.New("volume name starting with radix are not allowed")
+	ErrHostPathNotAllowed        = errors.New("HostPath is not allowed")
+)
```

Comment:
We usually put errors in own file. [Example](https://github.com/equinor/radix-api/blob/c4c7838709b911440766e04eaaef72f99a4c5476/api/alerting/errors.go)